docs/data/routes/docs/techniques/azure-deployment/en.md

```diff
@@ -78,6 +78,8 @@ The steps to configure an Azure server to accept a JSS app are mostly the same a
 
 ## Step 4: Deploy the app's files
 
+> This step is not necessary for the [Next.js SDK](../../nextjs/introduction/why-nextjs/en.md) or when using the [HTTP Rendering Engine](../../fundamentals/services/view-engine/en.md#http-rendering-engine).
+
```

Comment:
Do we explain why somewhere?

Reply:
added a bit on this, but otherwise covered in the 'differences' topic